_events/2022/2022-11-sc22-social.md

```diff
@@ -18,7 +18,8 @@ The Outreach working group has two questions for you:
 - Are you going to [SC22](https://sc22.supercomputing.org/), taking place this year in Dallas, TX?
 
 If you answered _Yes_ to either question, we invite you to join your fellow
-RSEs at a to-be-determined location for food, drinks, and fun.
+RSEs at the Exchange Hall at 6PM on Sunday for food, drinks, and fun.
+[https://theexchangehall.com/](https://theexchangehall.com/)
```

Comment:
```suggestion
RSEs at the [Exchange Hall](https://theexchangehall.com/)
at 6PM on Sunday for food, drinks, and fun.
```

Reply:
I always people will miss links so I repeated myself but happy to write it this way